build.sbt

```diff
@@ -20,9 +20,9 @@ resolvers ++= Seq(
   Resolver.jcenterRepo)
 
 lazy val AcceptanceTest = config("acceptance") extend Test
```

Comment:
any point in naming these tests "Component" like our other services? Means renaming the directory and no need for this val.